openid-federation-1_0.xml

```diff
@@ -5163,11 +5163,30 @@ Host: trust-anchor.example.com
                   a valid participant of the federation.
                   The HTTP response status code SHOULD be 401 (Unauthorized).
                 </t>
-                 <t hangText="invalid_issuer">
+		<t hangText="invalid_issuer">
                   <vspace/>
                   The endpoint cannot service the requested issuer.
                   The HTTP response status code SHOULD be 404 (Not Found).
                 </t>
+		<t hangText="invalid_subject">
+		  <vspace/>
+		  The endpoint cannot service the requested subject.
+		  The HTTP response status code SHOULD be 404 (Not Found).
+		</t>
+		<t hangText="invalid_trust_anchor">
+		  <vspace/>
+		  The Trust Anchor cannot be found or used.
+		  The HTTP response status code SHOULD be 404 (Not Found).
+		</t>
+		<t hangText="trust_chain_validation_failed">
```

Comment:
Missing description - add "The Trust Chain validation failed"?

Reply:
Added